pycbc/waveform/generator.py

```diff
@@ -34,6 +34,7 @@
 from pycbc.waveform.utils import apply_fd_time_shift, taper_timeseries
 from pycbc.detector import Detector
 import lal as _lal
+from pycbc import gate
```

Comment:
@cdcapano why do we have two gating functions in PyCBC? It looks like the ``pycbc.gate`` module used here is not the same as @titodalcanton's code to do [perform gating in ``pycbc.strain``](https://github.com/ligo-cbc/pycbc/blob/master/pycbc/strain.py#L801)

Reply:
@duncan-brown There isn't two different gating functions. Look at gate.py; it just contains convenience functions for loading gating options, and applying gates to time domain and frequency domain data. The actual gating is done by Tito's function; see [line 81 of gate.py](https://github.com/ligo-cbc/pycbc/blob/master/pycbc/gate.py#L81).

The history of this was these functions were originally convenience functions I had stuck at the top of `pycbc_inference` for applying gates to the analyzed data and the data used for PSD estimation. Chris then moved these to their own module; see PR #1358.

I suppose it would be good to move Tito's gate function over to gate.py, but I think it'd be best to do that after O2 is over. I'd also like to incorporate my windowing module at some point, so we can do more than just Tukey windows.